### PR TITLE
Local analysis playbook xsiam enhancements

### PR DESCRIPTION
**Pull request by efelmandar**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-3640)

## Description
Adding the following changes to improve playbook logic:
* Improved implementation by executing the WildFire verdict flip request after investigation phase
* Added a check for existing WildFire report before detonating the file
* Added an option to skip manual task and close false positive automatically by default

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml [hunk 1]
```diff
@@ -1093,7 +1126,7 @@ tasks:
           accessor: agentid
       timeRange:
         simple: 2 hours ago
-    separatecontext: false
+    separatecontext: true
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml:1129]: should be false

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml [hunk 2]
```diff
@@ -1567,8 +1605,164 @@ tasks:
...
+    isautoswitchedtoquietmode: false
+  "75":
+    id: "75"
+    taskid: a953691c-5334-47a3-821f-304f8ed1fbfd
+    type: condition
+    task:
+      id: a953691c-5334-47a3-821f-304f8ed1fbfd
+      version: -1
+      name: is Manual Review required?
+      description: "Should we wait for manual review"
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation.yml:1627]: "Checks whether the playbook should wait for manual review"

File: Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation_README.md [hunk 3]
```diff
@@ -72,6 +73,7 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 | AutoRecovery | Whether to execute the Recovery playbook. | False | Optional |
 | AutoCloseAlert | Whether to close the alert automatically or manually, after an analyst's review. | False | Optional |
 | ShouldRescanBenign | Whether to rescan \(Using WildFire detonate file\) benign files. | True | Optional |
+| ShouldManualReviewFP | Whether to ask for manual review before FP handling, should be True or False | False | Optional |
```
[the remark this answers is not in the document]
Comment (efelmandar) [reply Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation_README.md:76]: only in the description or in the inputs name as well?  ShouldManualReviewFalsePositive sounds a little long

[the remark this answers is not in the document]
Comment (altmannyarden) [reply Packs/Core/Playbooks/playbook-Local_Analysis_alert_Investigation_README.md:76]: Only in the description

**Review by altmannyarden — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Pull request merged**